docker-compose.yml

```diff
@@ -259,10 +259,10 @@ services:
   ##
   # Postgres database server
   db: &db
-    image: postgres:10-alpine
+    image: postgres:11-alpine
     networks:
       internal:
-  db-dev:
+  db-dev: &db-dev
```

Comment:
Haven't checked this change, is it necessary? Don't know enough about this syntax to call it

Reply:
Id like to leave this here. It helps to be able to spin up containers alone to check on their state.